Documentation/MSTest-Migration-EditorConfig.txt

```diff
@@ -0,0 +1,125 @@
+# Example .editorconfig for migrating from Philips MSTest analyzers to Microsoft's official MSTest analyzers
+# This configuration demonstrates the recommended migration settings
+
+# ==================================================
+# MICROSOFT MSTEST ANALYZERS (Recommended)
+# ==================================================
+# Install: <PackageReference Include="MSTest.Analyzers" Version="3.10.0" PrivateAssets="all" />
+
+# Core validation rules (high priority)
+dotnet_diagnostic.MSTEST0002.severity = warning  # TestClass should be valid
```

Comment:
This package tries to follow strict guidelines by default, rather than cluttering the output with suggestions and warnings. I was able to safely set all 50 MS analyzers to error, except for MSTEST0019. Let's do the same herein. Ie., people using this package already know it's strict by default, so they presumably may want the same strictness in the MS flavors.